frontend/app/src/components/defi/wizard/ModuleSelector.vue

```diff
@@ -89,16 +89,16 @@ const { activeModules } = storeToRefs(useGeneralSettingsStore());
 const { update: updateSettings } = useSettingsStore();
 
 const balancesStore = useNonFungibleBalancesStore();
-const { balances } = storeToRefs(balancesStore);
-const { fetchNonFungibleBalances } = balancesStore;
+const { fetchNonFungibleBalances, reset: resetNonFungibleBalances } =
+  balancesStore;
```

Comment:
Ok so the reset function is used here (btw if the comments seem to have a weird order is because I don't review from top to bottom)

Maybe we can use `$reset` here. I don't see why can't fully reset the store to default.